### PR TITLE
Python 3 compliance

### PR DESCRIPTION
**Pull request by jleveque**
As part of the migration from Python 2 to Python 3, this PR ensures that all Python files in sonic-utilities are Python 3-compliant, and work with both Python 2 and Python 3.

Once this is merged, we can begin building a Python 3-based version of the sonic-utilities package.

### CODE REVIEW

**Comment by lguohan** [comment]
not sure why the build fails here.

File: pfc/main.py [hunk 1]
```diff
@@ -110,9 +110,9 @@ def showPfcPrio(interface):
         table.append([intf, entry.get('pfc_enable', 'N/A')])
     
     sorted_table = natsorted(table)
-    print '\n'
-    print tabulate(sorted_table, headers=header, tablefmt="simple", missingval="")
-    print '\n'
+    click.echo('\n')
```
Comment (qiluo-msft) [inline pfc/main.py:113]:
```suggestion
    click.echo()
```

File: scripts/watermarkstat [hunk 2]
```diff
@@ -56,7 +56,7 @@ class Watermarkstat(object):
         def get_queue_type(table_id):
             queue_type = self.counters_db.get(self.counters_db.COUNTERS_DB, COUNTERS_QUEUE_TYPE_MAP, table_id)
             if queue_type is None:
-                print >> sys.stderr, "Queue Type is not available!", table_id
+                sys.stderr.write("Queue Type is not available in table '{}'".format(table_id))
```
Comment (qiluo-msft) [inline scripts/watermarkstat:59]: How about `print(*args, file=sys.stderr)`

Comment (jleveque) [reply scripts/watermarkstat:59]: I considered it, but this method results in a shorter line.

**Comment by jleveque** [comment]
@lguohan:

> not sure why the build fails here.

It looks like `config reload` is failing because the new port breakout functionality is trying to obtain the hwsku, but the database isn't populated: https://github.com/Azure/sonic-utilities/blob/master/config/main.py#L63

Not sure why we just started seeing this failure on this PR, though; it should have existed.

File: pfc/main.py (hunk 1 above)
Comment (jleveque) [reply pfc/main.py]: Changed all occurrences.

File: scripts/watermarkstat (hunk 2 above)
Comment (jleveque) [reply scripts/watermarkstat]: Changed all instsances.

**Pull request merged**